src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Great job. Everything looks perfect. Please kindly ensure that The value of action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

Reply:
Thank you @tufoinnkuo10 for reviewing my project, 
Thank you for your motivating feedback,

I have modified the value of check_status to follow the ducks pattern as suggested

THANKS